client/proposer/service_test.go

```diff
@@ -77,31 +114,20 @@ func TestProposerRoundTrip(t *testing.T) {
 		<-fakeProposer.msgChan
```

Comment:
Also remove this line of `<-fakeProposer.msgChan` , we dont need the receive from the channel. Could you also change the loop counter from 7 to 4

Reply:
Sure, done.